mkt/data/base-images/mysql/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mozillamarketplace/centos-python27-mkt:0.6
+FROM centos:centos6
 
 ADD yum/mysql.repo /etc/yum.repos.d/mysql.repo
-RUN yum install -y mysql-community-server mysql-community-devel
+RUN yum install -y mysql-community-server mysql-community-devel; yum -y clean all
```

Comment:
This would be clearer.

<pre><code>RUN yum install -y mysql-community-server mysql-community-devel \
    yum -y clean all</code></pre>